stage/one/s8_tuple.py

```diff
@@ -0,0 +1,38 @@
+from utils.form import blank_form
+from utils import fields_generate
+
+route = {
+    'type': blank_form,
+    'url': '/stage/one/tuple',
+    'methods': [ 'GET', 'POST' ]
+}
+data = {
+    'title': 'Introduction to Tuple',
+    'description': [
+        'Hello there!',
+        "Let's get started with tuples",
+        'Try to print a tuple with me'
+    ],
+    'code': [
+        'a=(1,"x","TUPLE")',
+        'print(a)'
+    ],
```

Comment:
[This part](https://github.com/YunOSC/PyFun-Backend/blob/dc4f7c8b7fc760184bac47c3ee3defb6c9d8b33a/stage/one/s8_tuple.py#L16,L19
) is wrong, because there is no any blank for beginners to fill in.
Maybe you misunderstand what is this repo for.

You can create some blank to fill in.